ci/build_alpine_container/Makefile

```diff
@@ -0,0 +1,34 @@
+PHONY: all
+
+ENVOY_TAG?="latest"
+ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+
+all: binary container test
+
+binary:
+	echo "CWD: ${ROOT_DIR}"
+	rm -rf ./bin
+	mkdir -p ./bin/stripped
+	mkdir -p ./bin/debug
+	mkdir -p ./bin/configs
+	docker run -t -i --name build-envoy-alpine-ct -v ${ROOT_DIR}/../../:/source lyft/envoy-build:latest /bin/bash -c \
+		"set -x && cd /source && /source/ci/do_ci.sh normal && \
```

Comment:
To make this run faster, can we do a server_only build. We don't need to build any of the tests or run them.